spaces/domains.json

```
@@ -827,5 +827,6 @@
   "vote.safuchain.net": "safuchain.eth",
   "vote.wandinvestments.com": "wandinvestments.com",
   "vote.lebo.finance": "lebo-finance.eth",
-  "vote.cirip.io": "cirip.io"
+  "vote.cirip.io": "cirip.io",
+  "vote.fndz.io": "fndzdao.eth",
```

Please delete the comma at the end of the line.
```suggestion
  "vote.fndz.io": "fndzdao.eth"
```